src/business-logic/services/interfaces/device-service.ts

```diff
@@ -0,0 +1,14 @@
+import { Device } from '../../../model/entities/device'
+import { INewsDevice } from '../../../model/entities/inews-device'
+import { TelemetricsDevice } from '../../../model/entities/telemetrics-device'
+
+export interface DeviceService {
+  readAllConfigurations(): Promise<(Device | INewsDevice | TelemetricsDevice)[]>
+  readConfiguration(deviceId: string): Promise<Device | INewsDevice | TelemetricsDevice>
+  create(config: Device | INewsDevice | TelemetricsDevice): Promise<void>
+  update(deviceId: string, config: Device): Promise<void>
+  delete(deviceId: string): Promise<void>
+  disconnect(deviceId: string): Promise<void>
+  connect(deviceId: string): Promise<void>
+  connectAll(): Promise<void>
```

Comment:
Proposed names/types:
```
getDevices(): Promise<Device[]>
getDevice(deviceId: string): Promise<Device>
connectAllDevices(): Promise<void>
```

I'm not sure you need the methods `create`, `update`, `delete`, `disconnect` and `connect` in this interface, but lets have a talk about it. It's too much for me to write :)